raiden/ui/prompt.py

```diff
@@ -36,6 +36,9 @@ def unlock_account_with_passwordfile(
 
     try:
         return account_manager.get_privkey(address_hex, password.strip())
+    except KeystoreFileNotFound as e:
+        click.secho(str(e), fg="red")
+        sys.exit(1)
     except ValueError:
```

Comment:
We use `ValueError` in a lot of places, which is thrown by enough different things that it can lead to choosing the wrong error message easily. On the bright side, it might also throw the right error messages in cases you didn't think about due to being rather generic. Generally I would use it more sparingly.

Reply:
What would you use more sparingly? Value Error? 100% ... This whole "bug" is due to overusing ValueError

Reply:
Yes, this is exactly what I was trying to say.